projects/example/src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import {Component} from '@angular/core';
 import {AppRatingComponent} from './rating.component';
-import {SmartTableColumnCustomType, SmartTableConfig} from 'projects/smart-table/src/lib/smart-table/smart-table.types';
+import {SmartTableColumnCustomType, SmartTableConfig} from 'projects/smart-table/src/lib/smart-table.types';
```

Comment:
Als ik me niet vergis importeer je library dependencies best zoals dat in externe projecten ook gaat gebeuren. Maw, een reference naar de "dist" folder waar het project inzit. Dit kun je doen door je project in de tsconfig file te configuereren en laten verwijzen naar de dist folder.

Ik ben er namelijk vrij zeker van dat de types van de smart-table niet geexporteerd worden (werden) waardoor je dit dus niet in je project kan doen. (Persoonlijk zou ik deze types ook exporteren moest dit nog niet gebeurd zijn)

CORRECT ME IF I'M WRONG

Reply:
Ja idd, dit kan beter; Maar de dependencies niet importeren vanuit de dist folder, anders verlies je live reloading. Je kunt in de tsconfig wél idd een alias opbouwen die dan uiteindelijk naar het project gaat verwijzen.